src/Illuminate/Auth/Passwords/PasswordBroker.php

```diff
@@ -112,7 +112,7 @@ public function emailResetLink(CanResetPasswordContract $user, $token, Closure $
         return $this->mailer->send($view, compact('token', 'user'), function ($m) use ($user, $token, $callback) {
             $m->to($user->getEmailForPasswordReset());
 
-            if (! is_null($callback)) {
+            if ($callback instanceof Closure) {
                 call_user_func($callback, $m, $user, $token);
```

Comment:
What if I'm sending `$callback = [$class, 'send'];`

This could potentially set a huge breaking BC.